src/Theme.re

```diff
@@ -100,6 +100,55 @@ let of_yojson = (~isDark=?, ~themeLoader, json: Yojson.Safe.t) => {
   parse(json);
 };
 
+module PlistDecoder = {
+  open Plist;
+
+  let repository = scopeName =>
+    assoc(
+      oneOf([
+        ("pattern", Pattern.of_plist(scopeName) |> map(pat => [pat])),
+        (
+          "patterns",
+          property("patterns", array(Pattern.of_plist(scopeName))),
+        ),
+      ]),
+    );
+
+  let theme = (~isDark=?, plist) => {
+    let%bind maybeType = option(property("type", string), plist);
+    let isDark = isDark |> Option.value(~default=maybeType == Some("dark"));
```

Comment:
I actually think the type property is non-standard (it's at least not mentioned here: https://www.sublimetext.com/docs/3/color_schemes_tmtheme.html), but can't see that it hurts.